pkg/cli/environment.go

```diff
@@ -95,6 +96,7 @@ func New() *EnvSettings {
 		RegistryConfig:   envOr("HYPPER_REGISTRY_CONFIG", hypperpath.ConfigPath("registry.json")),
 		RepositoryConfig: envOr("HYPPER_REPOSITORY_CONFIG", hypperpath.ConfigPath("repositories.yaml")),
 		RepositoryCache:  envOr("HYPPER_REPOSITORY_CACHE", hypperpath.CachePath("repository")),
+		HelmDriver:       os.Getenv("HELM_DRIVER"),
```

Comment:
:100: !

Reply:
ok, I don't like changing `HELM_DRIVER` to `HYPPER_DRIVER` anymore. If we change it, we need to also start overloading `ActionConfig.Init()` and wherever it gets used as `HELM_DRIVER`, to `HYPPER_DRIVER`. Example:

https://github.com/helm/helm/blob/6e4a61f21279560b4a4cf7b84088c9e7e75757a9/pkg/action/action.go#L411

Reply:
Yes but this calls of implementing #65 urgently so we do not miss any changes there....